src/models/User.js

```diff
@@ -0,0 +1,31 @@
+import axios from "axios";
+import { observable } from "mobx";
+
+const transport = axios.create({
```

Comment:
This is so we can use the cross-domain session from the client to the server. This is the companion piece fo the `cors` on the server.